node/src/Consumer.ts

```diff
@@ -50,6 +50,15 @@ export type ConsumerOptions =
 	 */
 	preferredLayers?: ConsumerLayers;
 
+	/**
+	 * Whether this Consumer should enable RTP retransmissions upon receipt of
+	 * RTCP NACK from the remote Consumer. If not set, it's by default true for
```

Comment:
Little clarification, this change would we also applicable to `consumer.rs`.
```suggestion
	 * Whether this Consumer should enable RTP retransmissions, storing sent RTP
	 * and processing the incoming RTCP NACK from the remote Consumer. 
	 ....
```